packages/utils/src/loader/svg-loader.ts

```diff
@@ -60,8 +67,19 @@ const isValidResponse = (response: XMLHttpRequest | undefined): response is XMLH
  */
 const extractSafeSVG = (response: XMLHttpRequest | undefined): SVGElement | null => {
   if (isValidResponse(response) && response.responseXML) {
-    const svgDocument = response.responseXML.cloneNode(true) as Document;
-    const svg = svgDocument.firstElementChild;
+    const svgDocument = response.responseXML;
+    /**
+     * Getting the first node of response that is an element,
+     * just in case when firstChild isn't SVG.
+     * ( XMLDOM `cloneNode()` and `firstElementChild` are not supported in IE11 )
+     */
+    let svg;
```

Comment:
I'm not sure, can we declare svg variable like this?
`const svg = svgDocument.childNodes.find(element => element. nodeType === Node.ELEMENT_NODE);`

Reply:
Not possible. childNodes isn't type array.

Reply:
can you just do: `const svg = svgDocument.children[svgDocument.children.length - 1]`?

Reply:
Updated